dialect_mysql.go

```diff
@@ -112,17 +113,17 @@ func (m *mysql) Destroy(s store, model *Model) error {
 	return nil
 }
 
-func (m *mysql) Delete(s store, model *Model, query Query) error {
-	sb := query.toSQLBuilder(model)
-
-	sql := fmt.Sprintf("DELETE FROM %s", m.Quote(model.TableName()))
-	sql = sb.buildWhereClauses(sql)
+var asRegex = regexp.MustCompile(`\sAS\s\S+`) // exactly " AS non-spaces"
 
-	_, err := genericExec(s, sql, sb.Args()...)
-	if err != nil {
-		return fmt.Errorf("mysql delete: %w", err)
-	}
-	return nil
+func (m *mysql) Delete(s store, model *Model, query Query) error {
+	sqlQuery, args := query.ToSQL(model)
+	// * MySQL does not support table alias for DELETE syntax until 8.0.
+	// * Do not generate SQL manually if they may have `WHERE IN`.
+	// * Spaces are intentionally added to make it easy to see on the log.
+	sqlQuery = asRegex.ReplaceAllString(sqlQuery, "  ")
```

Comment:
This one line is different from the generic version. It removes `AS alias` from the query.